regulations/views/preamble.py

```diff
@@ -172,6 +172,41 @@ def make_preamble_toc(nodes, depth=1, max_depth=3):
     ]
 
 
+def section_navigation(full_id, toc, depth=1):
+    toc_flat = [sect for sect in toc if sect.depth == depth]
```

Comment:
Isn't the preamble ToC nested? If so, we don't need the `depth` comparison, hey?